ex01.exs

```diff
@@ -28,6 +28,32 @@ defmodule Ex01 do
   """
   
   def counter(value \\ 0) do
```

Comment:
It took me a while to work out why you had the second clause in your `receive` expression. Then I saw you were using it to initialize the counter.

There's an easier way, though. The last parameter to `spawn` is a list of arguments to pass to the function that's being run. This means you could just do

~~~ elixir
count = spawn(Ex01, :counter, [ value ])
~~~

I'm not deducting anything because you came up with  a clever solution.